src/app/components/header/header.component.html

```diff
@@ -0,0 +1,25 @@
+  <mat-toolbar color="primary">
+  <div fxHide.gt-xs>
+      <button mat-icon-button (click)="onToggleSidenav()">
+          <mat-icon>menu</mat-icon>
+      </button>
+  </div>
+  <div>
+    <a href="#" class="navbar-brand">{{title}}</a>
```

Comment:
I think this class is from bootstrap and should be replaced.

Reply:
So looks like you use this in the css. I think this is leftover from the one we copied above so we should rename it to avoid conflicts.

Reply:
yea. i took out the icons. i'll clean it up but i think i'll make it show the typical 3 bar menu  icon.
as far as navbar, i'll do that.